webrender_api/src/display_list.rs

```diff
@@ -947,7 +947,7 @@ impl DisplayListBuilder {
         index
     }
 
-    fn push_item(&mut self, item: SpecificDisplayItem, info: &LayoutPrimitiveInfo) {
+    pub fn push_item(&mut self, item: SpecificDisplayItem, info: &LayoutPrimitiveInfo) {
```

Comment:
This should still be memory safe, just easier to panic by pushing unexpected items. So I think no `unsafe` is needed, just add some cautionary documentation on top.